Parent/Parent/Conversations/ComposeViewController.swift

```diff
@@ -44,9 +52,9 @@ class ComposeViewController: UIViewController, ErrorViewController {
         hiddenMessage: String?
     ) -> ComposeViewController {
         let controller = loadFromStoryboard()
+        controller.context = context
```

Comment:
this spot is much better ;)

Reply:
haha it has to be here because the next line triggers `viewDidLoad` which triggers `course?.refresh()` which triggers `lazy var course` which looks at `self.context` which was `nil` because it was getting set too late. ;)

Reply:
ah that makes sense 👍